java/client/src/org/openqa/selenium/By.java

```diff
@@ -44,6 +44,10 @@
  * </code>
  */
 public abstract class By {
+  static {
```

Comment:
we shouldn't add a bunch of static initializers 'everywhere' in the code. Probably just one would be good, during the construction of the 'RemoteWebDriver' class.